lib/lang.js

```diff
@@ -65,6 +65,8 @@ module.exports = {
     `This pull request was landed via ${landedRepos[repoName].sha}.`,
   error_approval: command =>
     `Unable to ${command}. At least one approved review is required.`,
+  error_up_to_date_passing: command =>
```

Comment:
tiny nit: the underscore is just after the type of comment it is, otherwise they're camel case